invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -553,9 +643,17 @@ def draw_circle(self, center, radius=2.5, width=2, line_colour=(255, 0, 0, 128),
         gc.FillPath(path)
         self._drawn = True
 
-        return (cx, -cy, radius*2, radius*2)
-
-    def draw_ellipse(self, center, width, height, line_width=2, line_colour=(255, 0, 0, 128), fill_colour=(0, 0, 0, 0)):
+        return (cx, -cy, radius * 2, radius * 2)
+
+    def draw_ellipse(
```

Comment:
center is tuple of two integers
radius is float
width is integer
height is integer
line_colour is tuple of 4 integers
fill_colour is tuple of 4 integers

Reply:
but `width` and `height` are annotated in the docstring as `float`:
```python
"""
Draw a ellipse centered at center with the given width and height.

Params:
    center: (x, y) position.
    width: ellipse width (float number).
    height: ellipse height (float number)
    line_width: line width.
    line_colour: RGBA line colour
    fill_colour: RGBA fill colour.
"""
```

Reply:
Sorry, you are correct.